src/js/view/utils/submenu-factory.js

```diff
@@ -12,7 +13,7 @@ const QUALITIES_SUBMENU = 'quality';
 const PLAYBACK_RATE_SUBMENU = 'playbackRates';
 const DEFAULT_SUBMENU = QUALITIES_SUBMENU;
 
-export const makeSubmenu = (settingsMenu, name, contentItems, icon) => {
+export const makeSubmenu = (settingsMenu, name, contentItems, icon, localizedText) => {
```

Comment:
I'd call `localizedText` something like `tooltipText`. I think that it's more clear to name the argument after how it will be used, rather than what it is. When I read it, I say it in my head as "the {argument}", like "the name", "the icon", etc. and "the tooltip text" sounds a lot more meaningful than "the localized text".